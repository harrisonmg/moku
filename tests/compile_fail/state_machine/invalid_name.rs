use moku::*;

#[state_machine(123)]
mod blinky {}

fn main() {}
