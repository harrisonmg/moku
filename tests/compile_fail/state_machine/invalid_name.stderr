error: expected identifier
 --> tests/compile_fail/state_machine/invalid_name.rs:3:17
  |
3 | #[state_machine(123)]
  |                 ^^^
